src/pages/Landing/components/EventsCalendarView.tsx

```diff
@@ -30,15 +29,42 @@ type AppointmentProps = {
 
 const Appointment = ({ children, data }: AppointmentProps) => {
   const theme = useTheme();
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+  const open = Boolean(anchorEl);
+
+  const handleClick = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorEl(anchorEl ? null : event.currentTarget);
+  };
+
+  const handleClickAway = () => {
+    setAnchorEl(null);
+  };
 
   const getColor = (event: EventCompact) =>
     theme.palette.colors[event.organizer?.slug.toLowerCase() === Groups.NOK.toLowerCase() ? 'nok_event' : 'other_event'];
   return (
-    <Link to={`${URLS.events}${data.id}/${urlEncode(data.title)}/`}>
-      <Appointments.Appointment data={data} draggable={false} resources={[]} style={{ backgroundColor: getColor(data as unknown as EventCompact) }}>
-        {children}
-      </Appointments.Appointment>
-    </Link>
+    <div>
+      <div onClick={handleClick}>
+        <Appointments.Appointment data={data} draggable={false} resources={[]} style={{ backgroundColor: getColor(data as unknown as EventCompact) }}>
+          {children}
+        </Appointments.Appointment>
+      </div>
+      <Popper
+        anchorEl={anchorEl}
+        modifiers={[
+          {
+            name: 'flip',
+            enabled: false,
+          },
+        ]}
+        open={open}
+        placement='top'
+        style={{ zIndex: 1000000 }}>
+        <ClickAwayListener onClickAway={handleClickAway}>
+          <div>{typeof data.id === 'number' ? <EventsCalendarPopover id={data.id} /> : null}</div>
```

Comment:
Vi vet at data.id faktisk alltid vil være et tall derfor kan du gjøre følgende (tror jeg)
```suggestion
          <EventsCalendarPopover id={Number(data.id)} />
```